scripts/get_refresh_token.sh

```diff
@@ -0,0 +1,9 @@
+curl -X POST https://visibility.amp.cisco.com/iroh/oauth2/token \
+    --header "Accept: application/json" \
+    --header "Content-Type: application/x-www-form-urlencoded; charset=UTF-8" \
+    --header "User-Agent: ob-http" \
+    --header "Authorization: Basic <BASE64 ENCODING OF CLIENT_ID:CLIENT_SECRET >" \
```

Comment:
It would be nice to be able to pass parameters to those commands.

I think we can use `openssl` to base64 encode any string.